Makefile

```diff
@@ -82,6 +82,7 @@ fmt:
 	@find . -name '*.go' -type f -not -path "*.git*" -not -name '*.pb.go' -not -name '*pb_test.go' | xargs gofmt -w -s
 	@find . -name '*.go' -type f -not -path "*.git*"  -not -name '*.pb.go' -not -name '*pb_test.go' | xargs goimports -w -local github.com/celestiaorg
 	@go mod tidy -compat=1.17
+	@cfmt -w -m=100 ./...
```

Comment:
we need to add this to `deps` right? so people can install it, otherwise they'll get errors

Reply:
We don't have `make deps`. Also note that we also have other tools like `markdownlint` and `golangci-lint` that would need to be downloaded via `deps`, but these all can be part of another issue and not necessarily this one.

Reply:
A bit skeptical though about choosing the way things are installed for users. E.g. `golangci-lint` can be installed through `apt/pacman`(`brew` for Linux) and installing it strictly by `go install` may be annoying.